mlops_ml_models/deploy_model_endpoint.py

```diff
@@ -21,6 +21,10 @@ def deploy_model(
         image_uri=(image_uri),  # The ECR image you pushed
         model_data=model_file,  # Location of your serialized model
         role=role,
+        env={
```

Comment:
By building these into the environment at runtime, we leave the door open for customers to reuse the same pycaret image across multiple models in their environment.